llvm/utils/lit/tests/Inputs/testrunner-custom-parsers/test.txt

```diff
@@ -4,6 +4,10 @@
 // MY_RUN: baz
 // MY_LIST: one, two
 // MY_LIST: three, four
+// MY_SPACE_LIST: orange
```

Comment:
We could add a test where we have multiple spaces between items.

Reply:
Great idea, pushed. This should be handled by my previous commit to avoid empty strings when splitting. (Without that logic, the initial space after the colon was emitting an empty string.)